cooperatives/cngis

```diff
@@ -0,0 +1,23 @@
+---
+title: County of Northumberland Geographic Information System
+parent: Cooperatives
+---
+
+# County of Northumberland Geographic Information System
+
+|                   |                                          |
+|:------------------|:-----------------------------------------|
+| model             | Collaborative Organizational Development
+| service type      | Information Technology
+| country           | United States
+| government members | Northumberland County, Environmental Systems Research Institute
```

Comment:
Based on the description, it looks like all counties in PA use this software. Since they're users and not necessarily members, it probably doesn't make sense to put "all PA counties" here. Instead, I think let's have a "states" row with PA in it, and change "government type" from "municipal" to "county"